src/Compilers/CSharp/Test/Emit3/Semantics/ExtensionTests.cs

```diff
@@ -17834,4 +17832,366 @@ static class E
         var format = new SymbolDisplayFormat(parameterOptions: SymbolDisplayParameterOptions.IncludeType | SymbolDisplayParameterOptions.IncludeModifiers);
         Assert.Equal("extension(ref readonly Int32)", symbol.ToDisplayString(format));
     }
+
+    [Fact]
+    public void SignatureConflict_01()
```

Comment:
nit: Consider adding an instance scenario where the extension parameters match, but the method's parameter has a different modifier.